docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -78,11 +78,15 @@ The following blocks are supported inside the definition of
 Hierarchy | Block | Description | Required
 --------- | ----- | ----------- | --------
 tls | [tls][] | Configures TLS for the HTTP client. | no
+metrics | [metrics][] | Configures the metrics that this component generates to monitor its state. | no
+resource_attributes | [resource_attributes][] | Configures the resource attributes that this component generates to annotate telemetry data. | no
```

Comment:
```suggestion
metrics | [metrics][] | Configures which metrics will be sent to downstream components. | no
resource_attributes | [resource_attributes][] | Configures resource attributes for metrics sent to downstream components. | no
```
`metrics` is not for the component's state :) It's the state of the vcenter endpoint.